source/_data/post_metadata.yml

```diff
@@ -0,0 +1,35 @@
+- title: "Training, the Simple way"
```

Comment:
Would like to better understand why we need to have a separate file to contain the metadata.
IMO, the metadata should exist in the frontmatter of the files which contain the posts.

Reply:
The posts (and frontmatter) are overwritten on deploy for blogs that are being actively aggregated. https://github.com/nilenso/blog.nilenso.com/blob/23061c3af58ae7106a9857696c28bfa79923ba45/planet.yml#L20

This means any changes in those posts will be lost on every deploy

Reply:
Ah okay, that information will definitely be lost every time there is a deploy. 

I am thinking we should move away from the aggregator model for a couple of reasons:

1. Different platforms (medium, personal blogs) have different schemas which gives this need of mapping their schema to the nilenso-blog schema.
2. Much better to have a central CMS rather than relying on the outside world. This gives us absolute control over our schema which can help us accommodate any new design changes in the future.

If we agree, we can take it up in a follow-up story.

Reply:
This is a larger conversation, can pick it up later. I think the reason we're using the aggregator model is so people can control, and own where they publish articles.

Using a central CMS would mean we'd either manually duplicate content, or that people would publish exclusively through nilenso. Not sure how people feel about this.